pom.xml

```diff
@@ -40,6 +40,7 @@
     <module>mongodb-plugins</module>
     <module>spark-plugins</module>
     <module>transform-plugins</module>
+    <module>googlecloud-plugins</module>
     <module>copybookreader-plugins</module>
```

Comment:
these two (`googlecloud-plugins`, `copybookreader-plugins`) should be added alphabetically, though perhaps leaving `hydrator-common` as the first